README.md

```diff
@@ -133,6 +133,36 @@ int main(void){
 }
 ```
 
+## Enabling traces in mbed OS 5
+
+To enable traces in mbed OS 5, add the following to your ``mbed_app.json``:
+
+```json
+{
+   "target_overrides": {
+      "*": {
+         "mbed-trace.enable": 1
```

Comment:
yes, `COMMON_PAL` feature is missing. this might change in future, but at the moment it is required.